tests/db/model/aspect/update.js

```diff
@@ -92,155 +88,136 @@ describe('db: aspect: update: ', () => {
     })
-    .catch((err) => done(err));
+    .catch(done);
   });
 
   it('update description of an aspect', (done) => {
     Aspect.findOne({ where: { name: u.name } })
     .then((o) => {
       const descriptionDefault = o.dataValues.description;
-      // console.log(o.dataValues.description);
       expect(o.dataValues).to.have.property('description')
       .to.equal(descriptionDefault);
       return o.update({ description: 'newDescription' });
     })
     .then((o) => {
       const newDefault = o.dataValues.description;
-      expect(o.dataValues).to.have.property('description').to.equal(newDefault);
-      // console.log(o.dataValues.description);
+      expect(o.dataValues).to.have.property('description', newDefault);
       done();
     })
-    .catch((err) => done(err));
+    .catch(done);
   });
 
   it('update helpEmail of an aspect', (done) => {
     Aspect.findOne({ where: { name: u.name } })
     .then((o) => {
       const helpEmailDefault = o.dataValues.helpEmail;
-      // console.log(o.dataValues.helpEmail);
       expect(o.dataValues).to.have.property('helpEmail')
-      .to.equal(helpEmailDefault);
+        .to.equal(helpEmailDefault);
       return o.update({ helpEmail: 'newemail@test.com' });
     })
     .then((o) => {
       const newDefault = o.dataValues.helpEmail;
-      expect(o.dataValues).to.have.property('helpEmail').to.equal(newDefault);
-      // console.log(o.dataValues.helpEmail);
+      expect(o.dataValues).to.have.property('helpEmail', newDefault);
       done();
     })
-    .catch((err) => done(err));
+    .catch(done);
   });
 
   it('update helpUrl of an aspect', (done) => {
     Aspect.findOne({ where: { name: u.name } })
     .then((o) => {
       const helpUrlDefault = o.dataValues.helpUrl;
-      // console.log(o.dataValues.helpUrl);
       expect(o.dataValues).to.have.property('helpUrl')
-      .to.equal(helpUrlDefault);
+        .to.equal(helpUrlDefault);
       return o.update({ helpUrl: 'http://www.updatedUrl.com' });
     })
     .then((o) => {
       const newDefault = o.dataValues.helpUrl;
       expect(o.dataValues).to.have.property('helpUrl').to.equal(newDefault);
-      // console.log(o.dataValues.helpUrl);
       done();
     })
-    .catch((err) => done(err));
+    .catch(done);
   });
 
   it('update isPublished of an aspect', (done) => {
     Aspect.findOne({ where: { name: u.name } })
     .then((o) => {
       const isPublishedDefault = o.dataValues.isPublished;
-      // console.log(o.dataValues.isPublished);
       expect(o.dataValues).to.have.property('isPublished')
-      .to.equal(isPublishedDefault);
+        .to.equal(isPublishedDefault);
       return o.update({ isPublished: false });
     })
     .then((o) => {
       const newDefault = o.dataValues.isPublished;
-      expect(o.dataValues).to.have.property('isPublished').to.equal(newDefault);
-      // console.log(o.dataValues.isPublished);
+      expect(o.dataValues).to.have.property('isPublished', newDefault);
```

Comment:
Since newDefault === o.dataValues. isPublished,
we can check for isPublished only
expect(o.dataValues).to.have.property('isPublished').

There's many cases of this in this test. It is quite redundant.